tests/src/Update.elm

```diff
@@ -110,6 +110,37 @@ all =
                     |> .present
                     |> Array.isEmpty
                     |> Expect.true "drawing should not be added to array"
+        , test "should cancel drawing for tiny free-draw drawing" <|
+            \() ->
+                model
+                    |> update (ChangeDrawing DrawFreeHand)
+                    |> update (StartDrawing start)
+                    |> update (FinishDrawing (shiftPosition 2 2 start))
+                    |> .edits
```

Comment:
I wrote a little helper function `getFirstAnnotation`. You could put that here and then do `Expect.equal Nothing`. But I don't have a preference :D